en/docs/install-and-setup/setup/security/configuring-transport-level-security.md

```diff
@@ -39,10 +39,35 @@ To prevent these types of security attacks, it is encouraged to disable the weak
 
 !!! info
     From **Firefox** 39.0 onwards, the browser does not allow to access Web sites that support DHE with keys less than 1023 bits (not just DHE\_EXPORT). 768/1024 bits are considered to be too small and vulnerable to attacks if the hacker has enough computing resources.
+    
 !!! tip
-    To use AES-256, the Java JCE Unlimited Strength Jurisdiction Policy files need to be installed. Download them from [http://www.oracle.com/technetwork/java/javase/downloads/index.html](index) .
-!!! tip
-    From Java 7, you must set the `jdk.certpath.disabledAlgorithms` property in the `<JAVA_HOME>/jre/lib/security/java.security` file to `jdk.certpath.disabledAlgorithms=MD2, DSA, RSA keySize < 2048` . It rejects all algorithms that have key sizes less than 2048 for MD2, DSA and RSA.
+    -   To use AES-256, the Java JCE Unlimited Strength Jurisdiction Policy files need to be installed. Download them from [http://www.oracle.com/technetwork/java/javase/downloads/index.html](index), if your java installation does not have it installed.
+
+#### Configuring Passthrough transport-level ciphers and TLS versions
```

Comment:
```suggestion
#### Configuring PassThrough transport-level ciphers and TLS versions
```
Is this the correct way of writing it?

Reply:
I can find both instances in our docs. I'll accept your suggestion and change. Thanks.